cookbook/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "module",
+  "dependencies": {
+    "@multiversx/sdk-core": "14.0.0",
```

Comment:
Without this file, and with adjusted imports - is it possible?

Reply:
Or, maybe, keep it, but use an alias from `"@multiversx/sdk-core"` to the local folder? And drop `sdk-dapp` :see_no_evil: